README.md

```diff
@@ -1,2 +1,5 @@
 aem-forms-samples
 =================
+
+# package-export-import
```

Comment:
Can we have more specific name for utility and folder since they are specific to cm and might confuse others